crates/orchestrator/src/queue/job_queue.rs

```diff
@@ -322,9 +370,48 @@ macro_rules! spawn_consumer {
 }
 
 pub async fn init_consumers(config: Arc<Config>) -> Result<(), JobError> {
-    spawn_consumer!(JOB_PROCESSING_QUEUE.to_string(), process_job, consume_job_from_queue, config.clone());
-    spawn_consumer!(JOB_VERIFICATION_QUEUE.to_string(), verify_job, consume_job_from_queue, config.clone());
+    spawn_consumer!(SNOS_JOB_PROCESSING_QUEUE.to_string(), process_job, consume_job_from_queue, config.clone());
+    spawn_consumer!(SNOS_JOB_VERIFICATION_QUEUE.to_string(), verify_job, consume_job_from_queue, config.clone());
+
+    spawn_consumer!(PROVING_JOB_PROCESSING_QUEUE.to_string(), process_job, consume_job_from_queue, config.clone());
+    spawn_consumer!(PROVING_JOB_VERIFICATION_QUEUE.to_string(), verify_job, consume_job_from_queue, config.clone());
+
+    spawn_consumer!(
+        DATA_SUBMISSION_JOB_PROCESSING_QUEUE.to_string(),
+        process_job,
+        consume_job_from_queue,
+        config.clone()
+    );
+    spawn_consumer!(
+        DATA_SUBMISSION_JOB_VERIFICATION_QUEUE.to_string(),
+        verify_job,
+        consume_job_from_queue,
+        config.clone()
+    );
+
+    spawn_consumer!(
```

Comment:
I am not sure but are we using proof_registration queues anywhere ? 
If not let's mark a todo instead of spawning them ? 
wdyt @byteZorvin

Reply:
Removed it